logger.Dockerfile

```diff
@@ -4,6 +4,9 @@ FROM redhat/ubi8-micro:latest
 ENV TINI_VERSION v0.19.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
 RUN chmod +x /sbin/tini
+
+# Non-root user
+USER 65532:65532
```

Comment:
Do we need this?

cass-operator already adds a pod security context by default:

```yaml
  securityContext:
    fsGroup: 999
    runAsGroup: 999
    runAsUser: 999
```

Reply:
Is `65532` a specific `uid`/`gid` that is required here, or just something that isn't `0`/root? If it doesn't matter, maybe go with `999` for consistency?

Reply:
We just want non-root, but is it even necessary to add the `USER` instruction since cass-operator sets it by default?

Reply:
Good point!  My understanding is that the container's `securityContext` will override the pod's `securityContext`.  And either of these will override the image's USER defined in the Dockerfile.

The question comes for k8ssandra related containers overall.  Do we want to provide group and user assignment at the securityContext only?  Speaking for our managed images in this case.

One thought is that if images are being utilized/executed (e.g. unit/integration tests) outside of an operator, a default user should still be applied at the Dockerfile level.  If we don't enforce the USER assignment as `non-root` in the docker file, this opens the image up to running as `root` in some cases.  Also, covers the case where there is a bug in the operator or there isn't security applied properly.  The image would still default to a non-root user.

Where I see this helping the most, is when used as a standalone test of the image, and this non-root default will prevent it from having root capabilities.  Otherwise, we may get false positives on the test runs due to root's capabilities for reading, writing, etc.

Reply:
@jsanda @emerkle826 
Are we good here with having the Dockerfile include the USER to protect us from defaulting to something that _could_ be less safe?

I'm o.k. with `cassandra` user since we are discussing the cass-operator, but the basic convention used in other images and [here](https://github.com/k8ssandra/cass-operator/blob/38f22c25fdb7f3c366adac06fb879f063e8ab797/config/manager/manager.yaml#L21) is the `65532` uid.

Reply:
The example you referenced is for the security context used for the operator pod itself. If we are going to add the `USER` instruction then I think it makes sense to set it to the cassandra uid since that it was it is actually set to by default.

Reply:
> The example you referenced is for the security context used for the operator pod itself. If we are going to add the `USER` instruction then I think it makes sense to set it to the cassandra uid since that it was it is actually set to by default.

:+1:

Reply:
Updated to cassandra:cassandra in the image